lib/server/server-main.js

```diff
@@ -2,3 +2,7 @@
 import './../imports/startup/server/server-index';
 
 import './../imports/api/documents/both/methods';
+
+import { ProseMeteorServer as ProseMeteorServerClass } from './../imports/api/prosemeteor/server/prosemeteor-server';
+
+export let ProseMeteorServer = ProseMeteorServerClass;
```

Comment:
@ejfrancis You can replace lines 6 and 8 with just the following instead of doing the rename dance. That's probably my fault since I did the same thing in the original repo. I think its clearer though to export directly.

`export { ProseMeteorServer } from './../imports/api/prosemeteor/server/prosemeteor-server';`